lib/membrane_h264_format/h264.ex

```diff
@@ -25,7 +25,7 @@ defmodule Membrane.H264 do
 
   For example, NTSC's framerate of ~29.97 fps is represented by `{30_000, 1001}`
   """
-  @type framerate_t :: {frames :: pos_integer, seconds :: pos_integer}
+  @type framerate_t :: {frames :: pos_integer, seconds :: pos_integer} | nil
```

Comment:
Would be nice to write what `nil` means - the framerate is unknown

Reply:
done